telegram/__init__.py

```diff
@@ -34,103 +34,103 @@
     ChatMemberLeft,
     ChatMemberBanned,
 )
-from .chatmemberupdated import ChatMemberUpdated
-from .chatpermissions import ChatPermissions
-from .files.photosize import PhotoSize
-from .files.audio import Audio
-from .files.voice import Voice
-from .files.document import Document
-from .files.animation import Animation
-from .files.sticker import Sticker, StickerSet, MaskPosition
-from .files.video import Video
-from .files.contact import Contact
-from .files.location import Location
-from .files.venue import Venue
-from .files.videonote import VideoNote
-from .chataction import ChatAction
-from .dice import Dice
-from .userprofilephotos import UserProfilePhotos
-from .keyboardbuttonpolltype import KeyboardButtonPollType
-from .keyboardbutton import KeyboardButton
-from .replymarkup import ReplyMarkup
-from .replykeyboardmarkup import ReplyKeyboardMarkup
-from .replykeyboardremove import ReplyKeyboardRemove
-from .forcereply import ForceReply
-from .files.inputfile import InputFile
-from .files.file import File
-from .parsemode import ParseMode
-from .messageentity import MessageEntity
-from .messageid import MessageId
-from .games.game import Game
-from .poll import Poll, PollOption, PollAnswer
-from .voicechat import (
+from ._chatmemberupdated import ChatMemberUpdated
+from ._chatpermissions import ChatPermissions
+from .files._photosize import PhotoSize
```

Comment:
the subpackages should also be private, e.g. `telgram._{files, games, inline, passport, payment}`. Let's skip `vendor` for now, because that contains a submodule. I don't know how well that plays with renaming and also `vendor` will likely be gone after the switch to asyncio, anyway …